test/Microsoft.AspNetCore.Server.KestrelTests/EngineTests.cs

```diff
@@ -548,9 +548,6 @@ public void ConnectionCanReadAndWrite(TestServiceContext testContext)
                         "POST / HTTP/1.1",
                         "Content-Length: 3",
                         "",
-                        "101POST / HTTP/1.1",
-                        "Content-Length: 3",
-                        "",
```

Comment:
101 is now closed after response so that made this test fail. That's why I added the new test.